src/plan/sticky/immix/global.rs

```diff
@@ -126,9 +126,6 @@ impl<VM: VMBinding> Plan for StickyImmix<VM> {
 
     fn release(&mut self, tls: crate::util::VMWorkerThread) {
         if self.is_current_gc_nursery() {
-            let was_defrag = self.immix.immix_space.release(false);
```

Comment:
We still need to call `immix_space.release()` here, right?

Reply:
Yes.  New objects can be allocated into the `immix_space` and the LOS, and we still need to release them.  I'll fix it.